engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/CurrentEngineVersionNode.java

```diff
@@ -0,0 +1,45 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.version.BuildVersion;
+
+@BuiltinMethod(
+    type = "Meta",
+    name = "engine_version",
+    description = "Returns the version of the currently running Enso engine.",
+    autoRegister = false)
+public class CurrentEngineVersionNode extends Node {
+
+  public Text execute() {
+    return getCurrentVersion();
+  }
+
+  @CompilerDirectives.TruffleBoundary
+  private Text getCurrentVersion() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("Enso Engine Version: ");
+    sb.append(BuildVersion.ensoVersion());
+    sb.append("\nDefault Edition: ");
```

Comment:
I am surprised that on your Windows this displays newlines? Shouldn't you rather use `System.lineSeparator()`?

Reply:
It works fine with `\n` and we use it all over the place, e.g. in displaying the Test results. So I'd prefer to keep it as is.